en/publish/publish-wechatgame.md

```diff
@@ -79,13 +79,13 @@ In addition to pure game content, the __WeChat__ game environment also provides
 5. Media: pictures, recordings, cameras, etc.
 6. Other: location, device information, scan code, NFC, etc.
 
-## Access to the subdomain of WeChat games
+## Access to the subdomain of WeChat Mini Games
 In order to protect its social relationship chain data, __WeChat__ Games has added the concept of subdomain, which is also called **Open Data Domain**, which is a separate game execution environment. The resources, engines, and programs in the subdomain are completely isolated from the main game. Developers can only access the __wx.getFriendCloudStorage()__ and __wx.getGroupCloudStorage()__ APIs provided by __WeChat__ in the subdomain to implement some rankings, for example.
 
-__Cocos Creator__ supports packaging to subdomains starting with __v1.9.1__. For details, please refer to [Access to the subdomain of WeChat game](../publish/publish-wechatgame-sub-domain.md).
+__Cocos Creator__ supports packaging to subdomains starting with __v1.9.1__. For details, please refer to [Access to the subdomain of WeChat Mini Game](../publish/publish-wechatgame-sub-domain.md).
 
 ## Reading
-- [WeChat game development documentation](https://mp.weixin.qq.com/debug/wxagame/dev/index.html)
+- [WeChat Mini Game development documentation](https://mp.weixin.qq.com/debug/wxagame/dev/index.html)
 - [WeChat Public Platform](https://mp.weixin.qq.com/)
 - [Small Program API Documentation](https://mp.weixin.qq.com/debug/wxadoc/dev/api/)
 - [WeChat Developer Tools Download](https://mp.weixin.qq.com/debug/wxagame/dev/devtools/download.html)
```

Comment:
These links have also been changed to the English version, please change it😊:
- WeChat Mini Game development documentation：https://developers.weixin.qq.com/miniprogram/en/dev/index.html
- WeChat Public Platform：https://mp.weixin.qq.com/?lang=en_US
- Small Program API Documentation: https://developers.weixin.qq.com/miniprogram/en/dev/api/index.html
- WeChat Developer Tools Download: https://developers.weixin.qq.com/miniprogram/en/dev/devtools/download.html
- WeChat Developer Tools Documentation: https://developers.weixin.qq.com/minigame/en/dev/devtools/devtools.html

Reply:
Yes, good thought, English links might be more readable to us English reading folks :-)